internal/getproviders/registry_client.go

```diff
@@ -52,7 +52,7 @@ var (
 
 func init() {
 	configureDiscoveryRetry()
-	configureRequestTimeout()
+	ConfigureProviderRequestTimeout()
```

Comment:
This doesn't look right to me: shouldn't be exported, and since it's an HTTP request timeout I'm not sure the provider infix helps.